tracing-subscriber/src/fmt/time/time_crate.rs

```diff
@@ -49,6 +49,23 @@ pub struct UtcTime<F> {
 
 #[cfg(feature = "local-time")]
 impl LocalTime<well_known::Rfc3339> {
+    /// Returns a formatter that formats the current [local time] in the
+    /// [RFC 3339] format (a subset of the [ISO 8601] timestamp format),
+    /// but with a variable number of decimal digits.
+    ///
+    /// Use [`LocalTime::iso_8601`] instead.
+    ///
+    /// [local time]: https://docs.rs/time/0.3/time/struct.OffsetDateTime.html#method.now_local
+    /// [RFC 3339]: https://datatracker.ietf.org/doc/html/rfc3339
+    /// [ISO 8601]: https://en.wikipedia.org/wiki/ISO_8601
+    #[deprecated(note = "this has a variable number of decimal digits, use `LocalTime::iso_8601()` instead")]
+    pub fn rfc_3339() -> Self {
+        Self::new(well_known::Rfc3339)
+    }
+}
+
+#[cfg(feature = "local-time")]
+impl LocalTime<well_known::Iso8601> {
     /// Returns a formatter that formats the current [local time] in the
     /// [RFC 3339] format (a subset of the [ISO 8601] timestamp format).
```

Comment:
this doc comment is incorrect, as this function returns the ISO 8601 formatter, rather than the RFC 3339 formatter:
```suggestion
    /// Returns a formatter that formats the current [local time] in the
    /// [ISO 8601] timestamp format.
```

Reply:
This doc comment is correct AFAICT, as RFC 3339 is a subset of ISO 8601 (as the comment says).